galaxy_ie_helpers/__init__.py

```diff
@@ -140,10 +140,26 @@ def get(dataset_id, history_id=None):
 
     return file_path
 
+def get_user_history (history_id=None):
+    """
+       Get all visible dataset infos of user history.
+       Return a list of dict of each dataset.
+       The last element is always None
+    """ 
+    history_id = history_id or os.environ['HISTORY_ID']
+    gi = get_galaxy_connection(history_id=history_id, obj=False)
+    hc = HistoryClient(gi)
+    history = hc.show_history(history_id, visible=True, contents=True)
+    # I add a None element, to be sure the list has at least 2 elements
+    # It's neccessary, in the case case where the list give to R, it will
+    # be interpreted as a list of list, and not only a list.
+    history.append(None)
```

Comment:
Can we hack around this on the R side? The None will be confusing I guess for other users of this function.